cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
-
+describe('Conduit', () => {
+  let user;
+  beforeEach(() => {
+    cy.visit('');
+    cy.task('generateUser').then((generateUser) => {
+      cy.log(generateUser);
+      user = generateUser;
+    });
+    cy.log(user);
   });
 
-  it('', () => {
-
+  it('should create article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('');
+    cy.get('.ion-compose').click();
+    cy.findByPlaceholder('Article Title').type('Test');
+    cy.findByPlaceholder("What's").type('Test1234');
+    cy.findByPlaceholder('Write').type('Test about test');
+    cy.findByPlaceholder('Enter').type(`Test666{enter}`);
+    cy.get('[type="button"]').click();
+    cy.visit('');
+    cy.get('.nav > :nth-child(2) > .link').click();
+    cy.get(':nth-child(2) > .preview-link > h1').should('be.visible');
```

Comment:
Let's check the article title here to be sure that the page with the created article is opened but not the random article 
```suggestion
    cy.get(':nth-child(2) > .preview-link > h1').should('be.visible');
```